app/views/posts/index.html.erb

```diff
@@ -5,6 +5,12 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="splashtitle">Let's Pair</h1>
                         <p class="slogantext">A collaborative community of developers who love to code.
+                        <br />
+                        <br />
+                        <%= link_to categories_path do %>
+                        <button type="button" class="btn btn-primary btn-lg">Find a Session</button>
```

Comment:
Minor note here, but indentation should have the button nested one level deeper:

```
<%= link_to categories_path do %>
      <button type="button" class="btn btn-primary btn-lg">Find a Session</button>
 <% end %>
```